Description: Use a signed char instead of a double for lab_struct
 The massive data structure was using a double for each struct member,
 which makes the resulting object extremely large. By switching it to a
 signed char we reduce the size from around 20 MiB to 2.5 MiB, so an order
 of magnitude. The new data type should be fine as the number literals do
 not exceed the type range.
 .
 In any case this gets embedded into at least three binaries, one of which
 contais an extremely generic filename(!). Which is still rather ridiculous
 as this should most possibly be computable at run-time, or stored in a
 shared data file, or in a single shared library or plugin.
Author: Guillem Jover <guillem@debian.org>
Origin: vendor
Bug-Debian:
Forwarded: no
Last-Update: 2014-07-18

---

--- lib/edgepaint/lab.h
+++ lib/edgepaint/lab.h
@@ -21,7 +21,7 @@ struct xyz_struct {
 typedef struct xyz_struct color_xyz;
 
 struct lab_struct {
-  double l, a, b;/* l: 0 to 100, a,b: -128 tp 128 */
+  signed char l, a, b;/* l: 0 to 100, a,b: -128 tp 128 */
 };
 typedef struct lab_struct color_lab;
 
